CHANGELOG.md

```diff
@@ -1,5 +1,184 @@
 # Changelog
 
+## [v4.7.1](https://github.com/honestbleeps/Reddit-Enhancement-Suite/releases/v4.7.1)
+
+Highlights:
+
+- Microsoft Edge Support (huge thanks @BenMcGarry and @erikdesjardins)
+- Massive infrastructure overhauls towards faster development, faster startup times (massive thanks @erikdesjardins, @mc10)
+- IndexedDB storage for Firefox: more stable than simple-storage (thanks @erikdesjardins)
+- Chrome storage improvements: increased performance and unlimited capacity (previously 5 MB) (thanks @erikdesjardins)
+- Show Images overhaul (huge thanks @larsjohnsen)
+- i.redd.it and redditmedia image support
+- A ridiculous amount of hard work from @erikdesjardins, @mc10, @larsjohnsen, @githue, @matheod, @thybag
+- Contributions from many other members in the community
+
+See [all pull requests merged](https://github.com/honestbleeps/Reddit-Enhancement-Suite/pulls?utf8=%E2%9C%93&q=merged%3A2015-12-05..2016-07-20%20base%3Amaster) for this version (300+) or browse by 
+[commit](https://github.com/honestbleeps/Reddit-Enhancement-Suite/compare/d35986d871577be6422b3ef445fb4f0c4bd30fcf...master) (1100+).
```

Comment:
We should be able to replace this URL with `https://github.com/honestbleeps/Reddit-Enhancement-Suite/compare/d35986d871577be6422b3ef445fb4f0c4bd30fcf...v4.7.1` now that 4.7.1 is tagged